apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -153,19 +178,13 @@ const CreateEventPage = ({ history }) => {
       <div className="page-container">
         <div className="create-event">
           <p className="create-event-text">Create Event</p>
-          <label htmlFor="imageFile" className="custom-file-upload event-input-border">
-            {imageFile ? (
-              <img src={imageFile} style={{ width: '50%', height: '50%' }} alt="Event" />
-            ) : (
-              <UploadIcon />
-            )}
-          </label>
-          <input
-            className="event-image-upload"
-            id="imageFile"
-            type="file"
-            onChange={handleFileUpload}
-          />
+          <CustomFileUpload
+            imageURL={imageURLFile}
+            hasErrors={formik.touched.imageFile && !!formik.errors.imageFile}
+            errorMessage={formik.errors.imageFile}
+            handleFileUpload={handleFileUpload}
+            uploadIcon={<UploadIcon />}
+          ></CustomFileUpload>
```

Comment:
```suggestion
          <CustomFileUpload
            imageURL={imageURLFile}
            hasErrors={formik.touched.imageFile && !!formik.errors.imageFile}
            errorMessage={formik.errors.imageFile}
            handleFileUpload={handleFileUpload}
            uploadIcon={<UploadIcon />}
          />
```
This component doesn't need children. So this syntax is cleaner.